diagnostic.md

```diff
@@ -21,13 +21,20 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --request GET http://www.example.com/donuts
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonut = function() {
```

Comment:
This is correct but don't forget to prevent your default actions by passing in the event, then preventing the default action.  event.preventDefault();